resources/patches/ungoogled-chromium/macos/fix-older-sdk-declarations.patch

```diff
@@ -0,0 +1,11 @@
+--- a/base/mac/sdk_forward_declarations.h
++++ b/base/mac/sdk_forward_declarations.h
+@@ -285,7 +285,7 @@
+ @end
+
+ // VNImageRequestHandler forward declarations.
+-typedef NSString* VNImageOption NS_STRING_ENUM;
++typedef NSString* VNImageOption;
```

Comment:
If you're worried about this breaking compatibility on newer systems, maybe we could use a compiler macro that goes by the SDK version?

Reply:
thing is it's already inside such macro